From bd38f0777f97224ceecf519dd0620d2f8a3839f2 Mon Sep 17 00:00:00 2001
From: omni <omni+alpine@hack.org>
Date: Sat, 18 Nov 2023 03:11:29 +0000
Subject: [PATCH] features: support zfs 2.2.x

ZFS has merged all but the spl kernel module into the zfs module

https://github.com/openzfs/zfs/commit/ad9e767657c3d3c0ec9e09a17f6732bcbf915401

These two modules now appear as

  /lib/modules/*/extra/spl.ko.gz
  /lib/modules/*/extra/zfs.ko.gz

This change allows for both pre and post ZFS 2.2.x kmod installations.
---
 features.d/zfs.modules | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/features.d/zfs.modules b/features.d/zfs.modules
index be5225a..d79cadd 100644
--- a/features.d/zfs.modules
+++ b/features.d/zfs.modules
@@ -1,7 +1,7 @@
 extra/avl
 extra/nvpair
-extra/spl
+extra/spl*
 extra/unicode
 extra/zcommon
-extra/zfs
+extra/zfs*
 extra/zpios
-- 
GitLab

